### ISSUE TITLE
Improve scrollbar UI

### ISSUE THREAD

**Issue by bvaughn** [facebook/react-devtools#1131]
Feedback from PR #1112

On OS X, a horizontal scrollbar covers some of the scrolling lists (e.g. snapshot selector, breadcrumbs strip) in a way that makes them difficult to use. Maybe there's a better UI we can use for this?

**Comment by bvaughn** [comment]
Yup, agreed. Not sure of a great alternative to this though given the limited space and the way scrollbars work. Open to suggestions.

**Comment by matthewbryancurtis** [comment]
+1 for the commits being blocked by the scrollbar. I don't know anything at all about developing for extensions, but I wonder if this is something that could be moved to a sidebar or a main panel? Which would allow it to be a little bigger?

**Comment by bvaughn** [comment]
v4 will fix this by removing that scrollbar entirely.

**Comment by bvaughn** [comment]
React DevTools has been rewritten and [recently launched a new version 4 UI](https://reactjs.org/blog/2019/08/15/new-react-devtools.html). The source code for this rewrite was done in a separate repository and now lives in the main React repo ([github.com/facebook/react](https://github.com/facebook/react/)).

Because version 4 was a total rewrite, and all issues in this repository are related to the old version 3 of the extension, I am closing all issues in this repository. If you can still reproduce this issue, or believe this feature request is still relevant, please open a new issue in the React repo: https://github.com/facebook/react/issues/new?labels=Component:%20Developer%20Tools
